### ISSUE TITLE
Enforce consistent ternary expressions

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1912]
# Rule request

## Thesis

We need to fix how we write ternary expressions, currently it has too many options:

```python
some = other if x > 1 else None
```

```python
some = (
    other 
    if x > 1 else None
)
```

```python
some = (
    other 
    if x > 1 
    else None
)
```

```python
some = (
    other if x > 1 
    else None
)
```

```python
some = other if x > 1 
    else None
```

Probably, there are other cases as well.

So, we should decide which ones we allow and which ones are going to be violations.
I know that single line ternary is always good.

To decide - we can analyze the code inside this project.

Related #1266 
Related #1706

**Comment by sobolevn** [comment]
I like this one:

```python
some = (
    other 
    if x > 1 
    else None
)
```

**Comment by sobolevn** [comment]
`ruff` now handles this.
